optimizer_wrapper.rb

```diff
@@ -491,16 +491,22 @@ def self.job_remove(api_key, id)
 
   def self.find_type(activity)
     if activity['service_id'] || activity['pickup_shipment_id'] || activity['delivery_shipment_id'] || activity['shipment_id']
-      'visit'
+      I18n.t('export_file.type_visit')
     elsif activity['rest_id']
-      'rest'
+      I18n.t('export_file.type_rest')
     elsif activity['point_id']
-      'store'
+      I18n.t('export_file.type_store')
     end
   end
 
+  def self.basic_header
+    [I18n.t('export_file.route'),
+     I18n.t('export_file.activity_id'), I18n.t('export_file.activity_point_id'), I18n.t('export_file.lat'), I18n.t('export_file.lon'), I18n.t('export_file.activity_type'), I18n.t('export_file.setup_duration'), I18n.t('export_file.duration'),
+     I18n.t('export_file.additional_value'), I18n.t('export_file.skills'), I18n.t('export_file.tags'), I18n.t('export_file.total_travel_time'), I18n.t('export_file.total_travel_distance')]
```

Comment:
```suggestion
    %w[
      route
      activity_id
      activity_point_id
      lat
      lon
      activity_type
      setup_duration
      duration
      additional_value
      skills
      tags
      total_travel_time
      total_travel_distance
    ].collect{ |label| I18n.t("export_file.#{label}") }
```

I think like this it is easier to add/remove labels and keep count of them and their order (otherwise it is easier to make mistakes while counting multiple items per line), what do you think?